### DISCUSSION TITLE
build minestom api

### DISCUSSION THREAD

**Discussion by g1444332** [Minestom/Minestom#2358] [Help] [answered]
Help please, I'm so dumb I can't build minestom api

**Comment by bivashy** [comment]
Please explain what you're trying to do, what problem you've encountered, what you've tried. 
Any details will be appreciated.

**Reply by g1444332** [reply]
I wanted to change the minestom api code a bit and when I changed it at build I got an error.
My actions:
1.git clone https://github.com/Minestom/Minestom.git
2.cd Minestom
3../gradlew build
Error:
/home/g1444332/IdeaProjects/Minestom/jcstress-tests/src/jcstress/java/net/minestom/server/tag/TagPathOverrideTest.java:3: error: package org.jglrxavpok.hephaistos.nbt does not exist
import org.jglrxavpok.hephaistos.nbt.NBT;
                                    ^

**Reply by bivashy** [reply, marked as answer]
> I wanted to change the minestom api code a bit and when I changed it at build I got an error. My actions: 1.git clone https://github.com/Minestom/Minestom.git 2.cd Minestom 3../gradlew build Error: /home/g1444332/IdeaProjects/Minestom/jcstress-tests/src/jcstress/java/net/minestom/server/tag/TagPathOverrideTest.java:3: error: package org.jglrxavpok.hephaistos.nbt does not exist import org.jglrxavpok.hephaistos.nbt.NBT; ^

This could be bad solution, but you can try to skip `jctress-tests`, `test` task like that:
`/gradlew build -x test -x :jcstress-tests:compileJcstressJava`

**Reply by DasBabyPixel** [reply]
Yeah I think this is a bug, hephaistos nbt was replaced with kyori nbt quite a while ago, jcstress-tests just never got updated
